Sources/Extensions/NotificationContent/MapViewController.swift

```diff
@@ -89,7 +89,14 @@ class MapViewController: UIViewController, NotificationCategory, MKMapViewDelega
 
         mapView.accessibilityIdentifier = "notification_map"
 
-        let span = MKCoordinateSpan(latitudeDelta: 0.1, longitudeDelta: 0.1)
+        let span: MKCoordinateSpan
+        if let latitudeDelta = CLLocationDegrees(templateValue: haDict["latitude_delta"]),
+           let longitudeDelta = CLLocationDegrees(templateValue: haDict["longitude_delta"]) {
+            span = MKCoordinateSpan(latitudeDelta: latitudeDelta, longitudeDelta: longitudeDelta)
+        } else {
+            span = MKCoordinateSpan(latitudeDelta: 0.1, longitudeDelta: 0.1)
+        }
```

Comment:
I'm fine with this how it is, though an alternate version could be:

```suggestion
        let span = MKCoordinateSpan(
            latitudeDelta: CLLocationDegrees(templateValue: haDict["latitude_delta"]) ?? 0.1,
            longitudeDelta: CLLocationDegrees(templateValue: haDict["longitude_delta"]) ?? 0.1
        )
```

if you feel like it might be worth including support for overriding only one of the deltas. At least in practice, this appears to have minimal effect.

Reply:
Yes, agree this is a bit nicer, updated!